ui/app/components/secret-edit.js

```diff
@@ -81,7 +87,36 @@ export default Component.extend(FocusOnInsertMixin, WithNavToNearestAncestor, {
     }
   },
 
-  waitForKeyUp: task(function*() {
+  waitForKeyUp: task(function*(name, value) {
+    // because path and key are not on the model performing custom validations instead of cp-validations
+    if (name === 'path') {
+      // no value indicates missing presence
+      !value
+        ? this.set('validationMessagePath', `Secret path can't be blank`)
```

Comment:
Rather than passing in each of these as separate values, what if we had a `validationMessages` object and then you could do `this.set('validationMessages.path', 'error message')`? Then we could access the object in the template

Reply:
Good call.  It was a bit of working around, and learned a good amount about the get helper in handlebars, but I got it.